wagtailio/project_styleguide/templates/patterns/includes/footer.html

```diff
@@ -1,3 +1,70 @@
 <footer class="footer">
     {% include "patterns/includes/menu_footer.html" %}
+    <div class="footer__base">
+        <div class="grid">
+            {# Links #}
+            <ul class="footer__links-wrap">
+                <li class="footer__link-item">
+                    <a class="footer__icon-link footer__icon-link--sponsor" href="/sponsors" aria-label="Sponsor Wagtail">
```

Comment:
Note: This is a direct URL, is the plan to make this a page?

Reply:
That's the plan - from the ticket:

> The Sponsor Wagtail link will go to a page we'll create on the new site with the URL "wagtail.org/sponsor".

Which has made me realise I need to remove an `s`!